<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="2" hostname="TR-JDGMXY2" failures="1" tests="1" name="HelpTest" time="0.052" errors="0" timestamp="2021-02-23T16:08:16 ART">
  <testcase classname="com.trgr.quality.maf.tests.HelpTest" name="RedirectedHelpLinkValidation" time="0.005">
    <failure type="java.lang.AssertionError" message="The following asserts failed:
	Exception in Test expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: The following asserts failed:
Exception in Test expected [true] but found [false]
at com.trgr.quality.maf.tests.HelpTest.RedirectedHelpLinkValidation(HelpTest.java:187)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 29 stack frames]]>
    </failure>
  </testcase> <!-- RedirectedHelpLinkValidation -->
  <testcase classname="com.trgr.quality.maf.tests.HelpTest" name="HelpLinkPresence" time="0.0">
    <ignored/>
  </testcase> <!-- HelpLinkPresence -->
  <testcase classname="com.trgr.quality.maf.tests.HelpTest" name="HelpLinkPresenceforAll" time="0.0">
    <ignored/>
  </testcase> <!-- HelpLinkPresenceforAll -->
</testsuite> <!-- HelpTest -->
